src/Ganeti/HTools/Node.hs

```diff
@@ -133,11 +134,12 @@ type TagMap = Map.Map String Int
 data Node = Node
   { name     :: String    -- ^ The node name
   , alias    :: String    -- ^ The shortened name (for display purposes)
-  , tMem     :: Double    -- ^ Total memory (MiB)
-  , nMem     :: Int       -- ^ Node memory (MiB)
-  , fMem     :: Int       -- ^ Free memory (MiB)
+  , tMem     :: Double    -- ^ Total memory (MiB) (state-of-world)
```

Comment:
Actually Ganeti doesn't have a state-of-record for total memory. AFAIK node size isn't stored in the database (config), so  the only way is to query the hypervisor, which is the state-of-world.